src/view/SignupView.java

```diff
@@ -112,16 +73,17 @@ public void keyPressed(KeyEvent e) {}
           @Override
           public void keyReleased(KeyEvent e) {}
         });
+
     // This makes a new KeyListener implementing class, instantiates it, and
     // makes it listen to keystrokes in the usernameInputField.
     //
     // Notice how it has access to instance variables in the enclosing class!
-    usernameInputField.addKeyListener(
```

Comment:
Why rename these?

Reply:
I was trying to make them more consistent with the button naming scheme, and it seemed clear enough that username was an input, but I'm happy to revert that change

Reply:
No, it's fine